README.md

```diff
@@ -1,24 +1,23 @@
-[![Build Status](https://travis-ci.com/IBM/summit-health-analytics.svg?branch=master)](https://travis-ci.com/IBM/summit-health-analytics)
+[![Build Status](https://travis-ci.com/IBM/example-health-analytics.svg?branch=master)](https://travis-ci.com/IBM/example-health-analytics)
 
-# Summit Health Analytics
+# Example Health Analytics
 
-This project is a conceptual Node.js analytics web application for a health records system, designed to showcase best in class integration of modern cloud technology, in collaboration with legacy mainframe code.
+This project is a conceptual Node.js analytics web application for a health records system, designed to showcase best in class integration of modern cloud technology, in collaboration with [legacy mainframe code](https://github.com/IBM/example-health-apis).
+> NOTE: This project is also compatible with the [Example Health JEE Application on Openshift](https://github.com/IBM/example-health-jee-openshift). See notes below for details.
 
-## Summit Health Context
+## Example Health Context
 
-Summit Health is a conceptual healthcare/insurance type company. It has been around a long time, and has 100s of thousands of patient records in a SQL database connected to a zOS mainframe.
+Example Health is a conceptual healthcare/insurance type company. It has been around a long time, and has 100s of thousands of patient records in a SQL database connected to a zOS mainframe. Their health records look very similar to the health records of most insurance companies.
 
-Summit's health records look very similar to the health records of most insurance companies.
-
-Here's a view a data analyst might see when they interact with the Summit Health Analytics Application:
+Here's a view a data analyst might see when they interact with the Example Health Analytics Application:
 
 ![](readme_images/screenshot.png)
 
-Summit has recently started understanding how data science/analytics on some of the patient records, might surface interesting insights. There is lots of talk about this among some of the big data companies.
+Example Health has recently started understanding how data science/analytics on some of the patient records, might surface interesting insights. There is lots of talk about this among some of the big data companies.
 
-Summit has also heard a lot about cloud computing. There is a lot of legacy code in the mainframe, and it works well for now, but Summit thinks it may be a complimentary opportunity to explore some data science/analytics in the cloud.
+Example Health has also heard a lot about cloud computing. There is a lot of legacy code in the mainframe, and it works well for now, but they think it may be a complimentary opportunity to explore some data science/analytics in the cloud.
```

Comment:
maybe switch up one of the uses of 'a lot' in these sentences?

Reply:
Updated